lighthouse-core/audits/screenshots.js

```diff
@@ -39,12 +39,6 @@ class Screenshots extends Audit {
    */
   static audit(artifacts) {
     const trace = artifacts.traces[this.DEFAULT_PASS];
-    if (typeof trace === 'undefined') {
```

Comment:
originally I was going to do that, but then noticed we have several audits (like `time-to-interactive`) that don't check that the trace exists before using, so even if it didn't fail here it would fail there. But ugh, you're probably right, the error message will be difficult to parse to figure out that you forgot to put a pass name in your custom config...